db/migrate/20200223032114_delete_smooch_bot_comments.rb

```diff
@@ -0,0 +1,5 @@
+class DeleteSmoochBotComments < ActiveRecord::Migration
+  def change
+  	 Rails.cache.write('check:migrate:delete_smooch_bot_comments', Time.now);
```

Comment:
Please fix the indentation and remove the `;` at the end of the line

Reply:
Done